definitions/output/reports/cwv_tech_categories.js

```diff
@@ -8,43 +8,45 @@ publish('cwv_tech_categories', {
 /* {"dataform_trigger": "report_cwv_tech_complete", "name": "categories", "type": "dict"} */
 WITH pages AS (
   SELECT
-    root_page,
+    client,
+    root_page AS origin,
     technologies
   FROM ${ctx.ref('crawl', 'pages')}
   WHERE
-    date = '${pastMonth}' AND
-    client = 'mobile'
+    date = '${pastMonth}'
     ${constants.devRankFilter}
 ), categories AS (
   SELECT
+    client,
     category,
-    COUNT(DISTINCT root_page) AS origins
+    COUNT(DISTINCT origin) AS origins
   FROM pages,
-    UNNEST(technologies) AS t,
-    UNNEST(t.categories) AS category
-  GROUP BY category
+    UNNEST(technologies) AS tech,
+    UNNEST(tech.categories) AS category
+  GROUP BY
+    client,
+    category
 ), technologies AS (
   SELECT
+    client,
     category,
     technology,
-    COUNT(DISTINCT root_page) AS origins
-  FROM pages,
-    UNNEST(technologies) AS t,
-    UNNEST(t.categories) AS category
-  GROUP BY
-    category,
-    technology
+    origins
+  FROM ${ctx.ref('reports', 'cwv_tech_technologies')}
+  WHERE client = 'mobile'
```

Comment:
Why mobile only?

Reply:
It's just a matter of adding a list of technology names per category.
Origins used to sort by popularity.

See `ARRAY_AGG(technologies.technology IGNORE NULLS ORDER BY technologies.origins DESC)` below.

Reply:
What if a technology is desktop only?

Reply:
Good point!

Reply:
Replaced it with total origins to consider cross-platform popularity.